dev-requirements-py2.txt

```
@@ -10,4 +10,4 @@ beautifulsoup4==4.8.1
 pypdf2==1.26.0
 requests-mock==1.7.0
 git+https://github.com/coyo8/parinx.git@6493798ceba8089345d970f71be4a896eb6b081d
-vulture==1.4
\ No newline at end of file
+vulture>=1.5
```

```suggestion
vulture==1.5
```
We should lock to a specific version